<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2496</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>2471</width>
      <height>951</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>FEFF</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_load_feff">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Load feff</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>441</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_chi_sim"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>60</y>
        <width>421</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_ft_sim"/>
     </widget>
     <widget class="QListWidget" name="listWidget_feff">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>341</width>
        <height>761</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>373</x>
        <y>420</y>
        <width>871</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_plot">
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_kweight_sim">
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>k weight</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_window_sim"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>window</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_kmin_sim">
         <property name="value">
          <double>3.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>kmin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_kmax_sim">
         <property name="value">
          <double>14.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>kmax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_make_ft_sim">
         <property name="text">
          <string>Make FT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_add_shells_to_fit">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>470</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add selected shells to fit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Feff">
     <attribute name="title">
      <string>EXAFS fit</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_chi"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>571</width>
        <height>461</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_ft"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>490</y>
        <width>511</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBox_window"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>window</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_kmin">
         <property name="value">
          <double>3.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>kmin</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox_kmax">
         <property name="value">
          <double>14.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>kmax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_make_ft">
         <property name="text">
          <string>Make FT</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>550</y>
        <width>2431</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_param">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_fit">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>490</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>490</y>
        <width>209</width>
        <height>36</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_load_chi">
         <property name="text">
          <string>Load chi</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_kweight">
         <property name="maximum">
          <number>4</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>kweight</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>490</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QSpinBox" name="spinBox_shells"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Shells</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
